Source/MediaInfo/Tag/File_Exif.cpp

```diff
@@ -1174,6 +1256,17 @@ void File_Exif::Get_IFDOffset(int8u KindOfIFD)
 //---------------------------------------------------------------------------
 void File_Exif::GetValueOffsetu(ifditem &IfdItem)
 {
+    auto GetDecimalPlaces = [](int32u denominator, int8u defaultDecimalPlaces = 3) -> int8u {
```

Comment:
I change my mind there, as there are so many different denominators, and you wish to have up to 9 decimals, it does not work well with e.g. 111. Let's do that if no problem seen on your side:

```suggestion
    auto GetDecimalPlaces = [](int32u denominator) -> int8u {
        int8u count = 0;

        while (denominator > 1) {
            ++count;
            denominator /= 10;
        }

        return count;
    };
```